packages/core/src/video/VideoForRendering.tsx

```diff
@@ -20,7 +21,9 @@ export const VideoForRendering: React.FC<RemotionVideoProps> = ({
 		if (!videoRef.current) {
 			return;
 		}
-		const frameInSeconds = frame / videoConfig.fps;
+		const frameInSeconds = FEATURE_FLAG_V2_BREAKING_CHANGES
+			? (frame + 1) / videoConfig.fps
+			: frame / videoConfig.fps;
```

Comment:
Let's make this change in the audio branch instead, then we won't have conflicts

Reply:
okay I'll revert this changes then.